proceeding/gammapy-icrc2017.tex

```diff
@@ -91,7 +91,7 @@ \section{Introduction}
 \label{sec:intro}
 
 % CTA science tools
-The Cherenkov Telescope Array (CTA) will observe the sky in very-high-energy
+The Cherenkov Telescope Array (CTA) will observe the sky in very high energy (VHE, E > 100$\,$GeV)
```

Comment:
Catherine commented: should be VHE, E > 20 GeV in CTA statements.
@joleroi - Can you update please?
(or should I take over)